src/components/ContributionsEpic.tsx

```diff
@@ -7,8 +7,7 @@ import { getCountryName, getLocalCurrencySymbol } from '../lib/geolocation';
 import { EpicTracking } from './ContributionsEpicTypes';
 import { ContributionsEpicReminder } from './ContributionsEpicReminder';
 import { Variant } from '../lib/variants';
-import { reminderJs } from './ContributionsEpic.js';
-import { EpicButtons } from './EpicButtons';
+import { ContributionsEpicButtons } from './ContributionsEpicButtons';
```

Comment:
Heh yeah similar to Andre's question above - is the renaming here intentional? It looks like the lost the `Contributions` prefix for the main epic component but gained it here?